internal/app/app.go

```diff
@@ -1208,6 +1208,19 @@ func (app *App) performSwitchover(clusterState map[string]*NodeState, activeNode
 		}
 		node := app.cluster.Get(host)
 		// in case node is a master
+
+		if app.config.ForceSwitchover {
+			err := node.SetOffline()
+			if err != nil {
+				return fmt.Errorf("switchover: failed to set node %s offline: %v", host, err)
+			}
+
+			err = node.SemiSyncDisable()
```

Comment:
Wrong order, I guess...
First we should disable semisync, so we won't hang on Wait for ACK.
And let's set old master online after it was successfully set read-only